plugins/inputs/cloudwatch/cloudwatch.go

```diff
@@ -153,22 +173,19 @@ func (c *CloudWatch) Gather(acc telegraf.Accumulator) error {
 
 func init() {
 	inputs.Add("cloudwatch", func() telegraf.Input {
-		return &CloudWatch{}
+		return &CloudWatch{
+			CacheTTL: "1hr",
```

Comment:
@joshhardy - this configures the default value to what it was before.
It's not required in the config file, if unspecified, it will be this.